README.md

```diff
@@ -36,39 +36,44 @@ Grant applications can be amended at any time. However, this _necessitates a ree
 
 ## :ballot_box_with_check: External Evaluations
 
-As part of the open grant program, we allow everyone to participate in the evaluation process of submitted milestones. Successfully merged external evaluations will receive **up to 1 KSM**, depending on the quality and depth of the evaluation. The KSM will be sent to their Kusama identity address.    
+As part of the open grant program, we allow everyone to participate in the evaluation process of submitted milestones. Successfully merged external evaluations will receive **up to 1 KSM**, depending on the quality and depth of the evaluation. The KSM will be sent to their Kusama identity address.
 
 ### Requirements
-* [On-Chain identity](https://guide.kusama.network/docs/en/mirror-learn-identity) on Kusama.
-* GitHub account.
-* The evaluation has to be submitted within 10 days after the milestone was delivered. 
+
+- [On-Chain identity](https://guide.kusama.network/docs/en/mirror-learn-identity) on Kusama.
+- GitHub account.
+- The evaluation has to be submitted within 10 days after the milestone was delivered.
 
 ### Process
+
 1. [Fork](https://github.com/w3f/Grant-Milestone-Delivery/fork) this repository.
 2. In the newly created fork, create a **copy** of the evaluation template ([`evaluations/evaluation-template.md`](evaluations/evaluation-template.md)).
 3. Name the file `project-name_milestone-number_your-github-username.md`.
 4. Fill out the template with the specifics of your evaluation. The higher the quality of your assessment and the level of detail, the higher the KSM recompense.
-5. Once you have completed the evaluation, click on "create new pull request".
+5. Once you have completed the evaluation, create a pull request.
 6. Add a comment or review to the pull request that contains the evaluated milestone delivery. This comment should contain a quick summary of your finding as well as link to the milestone submission.
 7. Milestone evaluations are double-checked by [W3F Grants Evaluators](https://github.com/w3f/Open-Grants-Program#w3f-grants-evaluators). Successful milestone evaluations are merged and the evaluator is rewarded with KSM.
 
-*In the future we plan to integrate a reputation system: with each successful milestone evaluation, the probability of being double-checked decreases. If you have done enough milestone evaluations, you become a recognised [W3F Grants Evaluator](https://github.com/w3f/Open-Grants-Program#w3f-grants-evaluators) yourself.*
+*You can evaluate as many deliveries as you like. With each successful evaluation, the probability of being double-checked decreases and once you have done enough of them, you'll be invited to become a [W3F Grants Evaluator](https://github.com/w3f/Open-Grants-Program#w3f-grants-evaluators) yourself.*
```

Comment:
@Noc2 is this okay?

Reply:
Maybe add something like you will be added to the readme as a grants evaluator or something like this. So people don’t assume they are hired as a web3 employee.

Reply:
I did mean to leave open the possibility that we would hire them if the evaluations are good, but it's true that it depends on other things as well and maybe it'd be promising too much